docs/reference-architectures/cratedb-azure-iot.rst

```diff
@@ -0,0 +1,27 @@
+====================
+CrateDB on Azure IoT
+====================
+
+Architecture
+============
+
+This reference architecture enables high throughput ingestion and enrichment
+of raw sensor data from a fleet of sensors. Once ingested into CrateDB, this
+data can be used to perform data science as well as build real-time data
+driven dashboards and reporting.
+
+.. _figure_1:
+
+.. figure:: cratedb-azure-iot.png
```

Comment:
i like the graphic, but the color of the markers seems a bit off. maybe you can reach out to @belringer to get a better color.

Reply:
Will do